libs/SalesforceSwiftSDK/SalesforceSwiftSDK/Extensions/SFSmartStoreExtensions.swift

```diff
@@ -190,7 +190,7 @@ extension SFSmartStore {
          - Returns: SFSoupSpec wrapped in a promise.
```

Comment:
Same question for SFSmartStoreExtensions.swift.
All smartstore operations are synchronous so why wrap them in promises?